### PR TITLE
remove spelled out CLI and CLI tools

### PR DESCRIPTION
**Pull request by mairaw**
CLI shouldn't be spelled out since it's a known industry term: https://styleguides.azurewebsites.net/StyleGuide/Read?id=2696&topicid=44109

Also, as discussed with @KathleenDollard, @wli3, and @tdykstra we're not calling CLI tools anymore. @natke there were two articles in ML.NET that could have similar changes done, but I haven't touched those.

### CODE REVIEW

File: docs/architecture/modern-web-apps-azure/common-client-side-web-technologies.md [hunk 1]
```diff
@@ -92,7 +92,7 @@ Components are defined using the @Component decorator function, which takes in m
...
 
-Angular also makes great use of command line interface (CLI) tooling. Getting started with Angular development locally (assuming you already have git and npm installed) consists of simply cloning a repo from GitHub and running `npm install` and `npm start`. Beyond this, Angular ships its own CLI tool which can create projects, add files, and assist with testing, bundling, and deployment tasks. This CLI tooling friendliness makes Angular especially compatible with ASP.NET Core, which also features great CLI support.
+Angular also makes great use of CLI tooling. Getting started with Angular development locally (assuming you already have git and npm installed) consists of simply cloning a repo from GitHub and running `npm install` and `npm start`. Beyond this, Angular ships its own CLI tool which can create projects, add files, and assist with testing, bundling, and deployment tasks. This CLI tooling friendliness makes Angular especially compatible with ASP.NET Core, which also features great CLI support.
```
Comment (tdykstra) [inline docs/architecture/modern-web-apps-azure/common-client-side-web-technologies.md]:
```suggestion
You can develop Angular apps by using a CLI. Getting started with Angular development locally (assuming you already have git and npm installed) consists of simply cloning a repo from GitHub and running `npm install` and `npm start`. Beyond this, Angular ships its own CLI, which can create projects, add files, and assist with testing, bundling, and deployment tasks. This CLI friendliness makes Angular especially compatible with ASP.NET Core, which also features great CLI support.
```

File: docs/core/about.md [hunk 2]
```diff
@@ -52,14 +52,14 @@ Multiple frameworks have been built on top of .NET Core:
...
 - The [ASP.NET Core runtime](https://github.com/dotnet/aspnetcore), which provides a framework for building modern cloud-based internet connected applications, such as web apps, IoT apps, and mobile backends.
-- The [.NET Core CLI tools](https://github.com/dotnet/sdk) and language compilers ([Roslyn](https://github.com/dotnet/roslyn) and [F#](https://github.com/microsoft/visualfsharp)) that enable the .NET Core developer experience.
-- The [dotnet tool](https://github.com/dotnet/core-setup), which is used to launch .NET Core apps and CLI tools. It selects the runtime and hosts the runtime, provides an assembly loading policy, and launches apps and tools.
+- The [.NET Core SDK](https://github.com/dotnet/sdk) and language compilers ([Roslyn](https://github.com/dotnet/roslyn) and [F#](https://github.com/microsoft/visualfsharp)) that enable the .NET Core developer experience.
+- The [dotnet tool](https://github.com/dotnet/core-setup), which is used to launch .NET Core apps and CLI commands. It selects the runtime and hosts the runtime, provides an assembly loading policy, and launches apps and tools.
```
Comment (tdykstra) [inline docs/core/about.md]: Should "dotnet tool" be "dotnet CLI command"?

File: docs/core/deploying/creating-nuget-packages.md [hunk 3]
```diff
@@ -5,7 +5,7 @@ author: cartermp
 ms.date: 06/20/2016
 ms.technology: dotnet-cli
 ---
-# How to create a NuGet package with .NET Core command-line interface (CLI) tools
+# How to create a NuGet package with .NET Core CLI
```
Comment (tdykstra) [inline docs/core/deploying/creating-nuget-packages.md]:
```suggestion
# How to create a NuGet package with the .NET Core CLI
```

File: docs/core/docker/introduction.md [hunk 4]
```diff
@@ -13,7 +13,7 @@ For more information about how to install Docker, see the download page for [Doc
 
 ## Docker basics
 
-There are a few concepts you should be familiar with. The Docker client has a command line interface program you use to manage images and containers. As previously stated, you should take the time to read through the [Docker overview](https://docs.docker.com/engine/docker-overview/) documentation. 
+There are a few concepts you should be familiar with. The Docker client has a CLI program you use to manage images and containers. As previously stated, you should take the time to read through the [Docker overview](https://docs.docker.com/engine/docker-overview/) documentation.
```
Comment (tdykstra) [inline docs/core/docker/introduction.md]:
```suggestion
There are a few concepts you should be familiar with. The Docker client has a CLI command you use to manage images and containers. As previously stated, you should take the time to read through the [Docker overview](https://docs.docker.com/engine/docker-overview/) documentation. 
```

File: docs/core/tools/dotnet.md [hunk 5]
```diff
@@ -1,6 +1,6 @@
 ---
 title: dotnet command
-description: Learn about the dotnet command (the generic driver for the .NET Core CLI tools) and its usage.
+description: Learn about the dotnet command (the generic driver for the .NET Core CLI) and its usage.
```
Comment (tdykstra) [inline docs/core/tools/dotnet.md]: Here it's called the dotnet command, not the dotnet tool

File: docs/core/tools/extensibility.md [hunk 6]
```diff
@@ -1,16 +1,16 @@
...
-# .NET Core CLI tools extensibility model
+# .NET Core CLI extensibility model
 
-This document covers the different ways you can extend the .NET Core Command-line Interface (CLI) tools and explain the scenarios that drive each one of them.
+This article covers the different ways you can extend the .NET Core CLI and explain the scenarios that drive each one of them.
 You'll see how to consume the tools as well as how to build the different types of tools.
 
-## How to extend CLI tools
-The CLI tools can be extended in three main ways:
+## How to extend CLI
```
Comment (tdykstra) [inline docs/core/tools/extensibility.md]:
```suggestion
## How to extend the CLI
```

File: docs/core/tutorials/libraries.md [hunk 7]
```diff
@@ -1,12 +1,12 @@
 ---
-title: Developing Libraries with Cross Platform Tools
-description: Learn how to create .NET Core libraries using the .NET Core CLI tools. You'll create a library that supports multiple frameworks.
+title: Develop libraries with .NET Core CLI
```
Comment (tdykstra) [inline docs/core/tutorials/libraries.md]:
```suggestion
title: Develop libraries with the .NET Core CLI
```

File: docs/core/tutorials/libraries.md [hunk 8]
```diff
@@ -1,12 +1,12 @@
...
+description: Learn how to create .NET Core libraries using the .NET Core CLI. You'll create a library that supports multiple frameworks.
 author: cartermp
 ms.date: 05/01/2017
 ---
-# Develop libraries with cross-platform tools
+# Develop libraries with .NET Core CLI
```
Comment (tdykstra) [inline docs/core/tutorials/libraries.md]:
```suggestion
# Develop libraries with the .NET Core CLI
```

File: docs/csharp/tutorials/console-teleprompter.md [hunk 9]
```diff
@@ -9,7 +9,7 @@ ms.assetid: 883cd93d-50ce-4144-b7c9-2df28d9c11a0
 
 This tutorial teaches you a number of features in .NET Core and the C# language. You'll learn:
 
-- The basics of the .NET Core Command Line Interface (CLI)
+- The basics of .NET Core CLI
```
Comment (tdykstra) [inline docs/csharp/tutorials/console-teleprompter.md]:
```suggestion
- The basics of the .NET Core CLI
```

**Review by tdykstra — APPROVED** [review]
Looks good, just a few nit suggestions

**Comment by gewarren** [comment]
I'm not sold on this one. I see conflicting guidance about spelling acronyms out on first use - https://styleguides.azurewebsites.net/Styleguide/Read?id=2700&topicid=42256. To me, CLI is not that ubiquitous, and before joining this team last fall, I wouldn't have immediately known what it was.

**Review by tdykstra — COMMENTED** [review]
There are no doubt plenty of references to the CLI in the ASP.NET Core docs as well.

File: docs/core/docker/introduction.md (hunk 4 above)
Comment (mairaw) [reply docs/core/docker/introduction.md]: Wondering if we should just remove the noun here.

**Comment by gewarren** [comment]
Also:

https://docs.microsoft.com/en-us/cli/azure/?view=azure-cli-latest
https://aws.amazon.com/cli/

**Comment by mairaw** [comment]
I could use it once in the CLI overview, but the same guidance is there for Azure CLI: https://styleguides.azurewebsites.net/StyleGuide/Read?id=2696&topicid=44108

**Comment by gewarren** [comment]
> I could use it once in the CLI overview, but the same guidance is there for Azure CLI: https://styleguides.azurewebsites.net/StyleGuide/Read?id=2696&topicid=44108

I think it would be good to at least use it in the overview.

**Comment by mairaw** [comment]
You're right @gewarren. I'll fix that and also review that image that also says tools on the article you referenced 😢

**Comment by gewarren** [comment]
I realize it's unrelated to your PR, but seemed important and since you are in there anyway...

**Review by Thraka — APPROVED** [review]
Love it!

File: docs/core/toc.yml [hunk 10]
```diff
@@ -151,7 +151,7 @@
 - name: Application deployment
   href: deploying/index.md
   items:
-  - name: Deploy apps with CLI tools
+  - name: Publish apps with the CLI
```
Comment (tdykstra) [inline docs/core/toc.yml]: Line 158 needs updating too

**Comment by mairaw** [comment]
I think I've addressed all the comments so far, I appreciate all the reviews.

**Pull request merged**